e2e/pages/enterChildren/enterChildren.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
 
   enterChildDetails(name, day, month, year, gender = 'Boy') {
     I.fillField(this.fields(activeChild).fullName, name);
+    I.wait(2);
```

Comment:
Wait does remove problem. The real issue is that page object is wrongly stateful. Either make it stateless or maintain state better to accommodate retries.